test/parallel/test-net-better-error-messages-port-hostname.js

```diff
@@ -3,12 +3,18 @@ const common = require('../common');
 const net = require('net');
 const assert = require('assert');
 
-const c = net.createConnection(common.PORT, '***');
+const s = net.createServer();
+s.listen(0);
+const port = s.address().port;
```

Comment:
@Trott If that's the case, I would prefer using common.PORT instead rather.
I figured I'd stick with 0 cause like you said `the actual connection never happens` but I'm okay going either way.